translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml

```diff
@@ -1,201 +1,254 @@
     related:
-        heading: __post.related.heading
-        heading_tags: __post.related.heading_tags
-        heading_ecosystems: __post.related.heading_ecosystems
-        heading_groups: __post.related.heading_groups
-        heading_specifications: __post.related.heading_specifications
+        heading: 相关内容
+        heading_tags: 标签
+        heading_ecosystems: 生态系统
+        heading_groups: 组
+        heading_specifications: 技术标准
     feed:
-        title: __post.feed.title
-        link: __post.feed.link {feed_type}
+        title: 相关的 RSS
+        link: 订阅我们的{feed_type}推荐页 (TODO: feed翻译成 推荐页/信息源/聚合页?)
     comments:
-        count: __post.comments.count <span>{count}</span>
-        skip_to_form: __post.comments.skip_to_form
-        reply: __post.comments.reply <span class="visuallyhidden">{name}</span>
-        reply_attr: __post.comments.reply_attr {name}
+        count: 第<span>({count})</span>条评论
+        skip_to_form: 跳跃至评论区
+        reply: 回复<span class="visuallyhidden">{name}</span>
+        reply_attr: 回复{name}
         form:
-            title: __post.comments.form.title
-        closed: __post.comments.closed
+            title: 发布新评论
+        closed: 该贴的评论区已关闭。
 notes:
     info:
-        default_title: __notes.info.default_title
+        default_title: 信息
     successes:
-        default_title: __notes.successes.default_title
+        default_title: 成功
     warnings:
-        default_title: __notes.warnings.default_title
+        default_title: 警示 (TODO: Warning)
     errors:
-        default_title: __notes.errors.default_title
+        default_title: 出现了一些问题  #TODO: There is a problem
 events:
     # same_day = 1 when start_date == end_date
     date_range: >-
         {same_day, select,
             1  {{start_date}, {start_time} - {end_time} <b>{timezone}</b>}
-            other {{start_date}, {start_time} - {end_date}, {end_time} <b>{timezone}</b>}
+            其它 {{start_date}, {start_time} - {end_date}, {end_time} <b>{timezone}</b>}
         }
-    add_to_calendar: __events.add_to_calendar
-    event_details: __events.event_details
-    date: __events.date
-    location: __events.location
-    speakers: __events.speakers
-    host: __events.host
-    category: __events.category
-    website: __events.website
+    add_to_calendar: 添加到日历
+    event_details: 事件详情
+    date: 日期
+    location: 地点
+    speakers: 发言人
+    host: 主办方 # TODO: host?
+    category: 分类
+    website: 网站
 alumni:
-    date_range: __alumni.date_range {start} {end}
+    date_range: "{start} 到 {end}"
 error:
-    title: __error.title {status_code} {status_text}
-    intro: __error.intro {status_code} {status_text}
-    request_id: __error.request_id {request_id}
-    contact: __error.contact {params}
-    faq: __error.faq
+    title: 错误{status_code} - {status_text}
+    intro: >-
+        <p>服务器返回"{status_code} - {status_text}"错误。</p>
+    request_id: >-
+        <p>请求ID: {request_id}</p>
+    contact: __error.contact {params}   #TODO: not in .en file
```

Comment:
Hi, @jean-gui . `error.contact` and `error.faq` don't exist in the -en file. Should I remove them?

Reply:
Yes, they can very likely be removed.

Reply:
removed